README.md

```diff
@@ -159,10 +159,10 @@ Exclude dotfiles from directory listings with `opts.showDotfiles === false`. Def
 If showDir is enabled, add human-readable file sizes. Defaults to **true**.
 Aliases are `humanreadable` and `human-readable`.
 
-### `opts.hidePermissions`
-### `--hide-permissions`
-If hidePermissions is enabled, file permissions will not be displayed. Defaults to **false**.
-Aliases are `hidepermissions` and `hide-permissions`.
+### `opts.selectDisplay`
```

Comment:
Kind of funky naming. It sounds like English isn't your first language so that's fine. I'll try to brainstorm something I like. :)

Reply:
Yeap, you are right :)